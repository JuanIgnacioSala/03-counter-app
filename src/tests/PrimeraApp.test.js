import { getByText, render } from "@testing-library/react"
import PrimeraApp from "../PrimeraApp"
import React from 'react';
import {shallow} from 'enzyme'

describe('Pruebas en PrimeraApp',()=>{
    // test('Debe mostrar el mensaje "Hola, saludo"', () => {
    //     const saludo = "Hola, saludo";
    //     const {getByText} = render(<PrimeraApp saludo={saludo}/>);
    //     expect(getByText(saludo)).toBeInTheDocument();


    // })
    test('Debe mostrar <primerapp/> correctamente ', () => {
        
        const saludo = 'Hola, saludo'
        const wrapper = shallow(<PrimeraApp saludo={saludo}/>)

        expect(wrapper).toMatchSnapshot();

    })

    test('Mostrar subtitulo', () => {

        const saludo = 'Hola, saludo'
        const subtitulo = "Subtitulo aca abajo";
        const wrapper = shallow(<PrimeraApp saludo={saludo} subtitulo={subtitulo}/>)
        const textoParrafo = wrapper.find('p').text();
        expect(textoParrafo).toBe(subtitulo);

    })
    
    
})