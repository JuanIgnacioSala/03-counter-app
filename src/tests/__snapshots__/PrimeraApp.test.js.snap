// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en PrimeraApp Debe mostrar <primerapp/> correctamente  1`] = `
<Fragment>
  <h1>
    Hola, saludo
  </h1>
  <p>
    Subtitulo aca abajo
  </p>
</Fragment>
`;
